bears/general/CPDBear.py

```diff
@@ -41,7 +41,7 @@ class CPDBear(GlobalBear):
     def check_prerequisites(cls):  # pragma: no cover
         if which("bash") is None:
             return "bash is not installed."
-        elif which("run.sh") is None:
+        if which("pmd") is None and which("run.sh") is None:
```

Comment:
Sorry, I don't really get this. Is `PMD` essential for CPDBear to run? Or only on MacOS? Because if it's not and the user has `run.sh` but not `PMD`, your method won't work because of the `and`